benefits/admin.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from django.contrib import admin
+from django.contrib.auth.models import Group
+from django.template.response import TemplateResponse
+from benefits.core.models import TransitAgency
+from benefits.core import session
+
+
+class BenefitsAdminSite(admin.AdminSite):
+
+    def index(self, request, extra_context=None):
+        """
+        Display the main admin index page, which lists all of the installed
+        apps that have been registered in this site.
+        """
+        app_list = self.get_app_list(request)
+
+        context = {
+            **self.each_context(request),
+            "title": self.index_title,
+            "subtitle": None,
+            "app_list": app_list,
+            **(extra_context or {}),
+        }
+
+        request.current_app = self.name
+
+        staff_group = Group.objects.get(name=settings.STAFF_GROUP_NAME)
+        agency = TransitAgency.for_user(request.user)
```

Comment:
Might this fail for a superuser or Cal-ITP staff that doesn't have an associated transit agency? Seems like it should be wrapped under the `if` below.

Reply:
Yep, that's a good point. I moved it under the `if` in 772f369554c6e8eded3cb4eb65913ed93ba9e8a5